crates/core/component/dex/src/component/position_manager/counter.rs

```diff
@@ -24,13 +25,45 @@ pub(crate) trait PositionCounter: StateWrite {
             return 0;
         };
 
-        // This is safe because we only increment the counter via a [`Self::increase_position_counter`].
+        // This is safe because we only increment the counter via [`Self::increase_position_counter`].
```

Comment:
would the extra two bytes for a u32 really kill us here? we spend more than that on a dex prefix. what if we want to have more than 100k positions? or what if the limit is something close, like 60k, and a bunch of new positions are opened within the block, and then there's an overflow (since we allow the position count to float above the limit during the block before closing positions to restore it)

Reply:
Good point, there wasn't a great reason to set it to two bytes